src/components/modals/AddEditModal.js

```diff
@@ -0,0 +1,195 @@
+import React, { forwardRef, useMemo, useState } from 'react';
+import ReactModal from 'react-modal';
+import { useModal } from 'react-modal-hook';
+import Select from 'react-select';
+import ReactDatePicker from 'react-datepicker';
+import styled from 'styled-components';
+import {
+  modalStyles,
+  PrimaryButtonOutlined,
+  PrimaryButtonFilled,
+  CloseIcon,
+  ModalActions,
+} from '../../shared/modalstyles';
+
+const LeftContainer = styled.div`
```

Comment:
Can you try to add a file inside modals folder, move the styles to that folder and import them here?
Check how this works and implement this throughout the application.